powerauth-nextstep/src/main/java/io/getlime/security/powerauth/app/nextstep/configuration/NextStepServerConfiguration.java

```diff
@@ -252,4 +305,34 @@ public Audit audit() {
         return auditFactory.getAudit();
     }
 
+    /**
+     * Configure LDAP context source
+     * @return LDAP context source.
+     */
+    @ConditionalOnProperty(prefix = "powerauth.nextstep.ldap", name = "enabled", havingValue = "true")
+    @Bean
+    public LdapContextSource contextSource() {
+        LdapContextSource cs = new LdapContextSource();
+        cs.setUrl(ldapUrl);
+        cs.setBase(ldapBase);
+        if (!managerDn.isBlank()) cs.setUserDn(managerDn);
```

Comment:
Please do not write a one-line block without curly brackets.
```suggestion
        if (!managerDn.isBlank()) {
            cs.setUserDn(managerDn);
        }
```